src/domain/app/api/version.ts

```diff
@@ -0,0 +1,32 @@
+
+const Version = () => {
+    // const packageVersion = process.env.APP_VERSION ?? '';
+    // const release = process.env.NEXT_PUBLIC_RELEASE ?? '';
+    // const commitHash = process.env.NEXT_PUBLIC_COMMITHASH ?? '';
+    // const buildTime = process.env.BUILD_TIME ?? '';
+
+    // res.status(200).json({
+    //   status: 'ok',
+    //   release,
+    //   packageVersion,
+    //   commitHash,
+    //   buildTime,
+    // });
+
```

Comment:
IMHO Generally if code is commented out it should be removed—of course there are exceptions but I don't see any clear reason for keeping this code. If there is a very good reason for keeping this commented out code it should IMHO be given in comments as well.

Reply:
removed